src/main/java/com/project/gamestore/mappers/ReviewMapper.java

```diff
@@ -0,0 +1,56 @@
+package com.project.gamestore.mappers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.project.gamestore.dtos.ReviewApi;
+import com.project.gamestore.dtos.ReviewDTO;
+import com.project.gamestore.entities.Game;
+import com.project.gamestore.entities.Review;
+import com.project.gamestore.entities.User;
+import com.project.gamestore.repositories.GameRepository;
+import com.project.gamestore.repositories.UserRepository;
+
+@Component
+public class ReviewMapper {
+
+    @Autowired
+    public UserMapper userMapper;
```

Comment:
The should be declared `private` since you shouldn't be able to modify them externally, only to make use of them through methods. Please also check in other services/controllers if you have fields declared `public` and please change them to `private`. Maybe I've missed them in earlier PRs. Sorry for not noticing them earlier.